lib/ethereum-address.spec.js

```diff
@@ -142,4 +143,23 @@ describe('EthereumAddress', () => {
             expect(EthereumAddress.from(undefined)).to.be.null;
         });
     });
+
+    describe('#assertInstance()', () => {
+
+        context('EthereumAddress instance', () => {
+            it('does not throw error', () => {
```

Comment:
This test should probably be moved up to the `verifyEthereumAddressInstance` function as it is a requirement for all scenarios where an address has been validly constructed